peppy/sample.py

```diff
@@ -302,6 +302,16 @@ def project(self):
         """
         return self[PRJ_REF]
 
+    @property
+    def sample_name(self):
```

Comment:
Should it be named like `sample_name` or should it be different property name
Do you have a suggestion @nsheff ?